src/libstd/sys/unix/tty.rs

```diff
@@ -42,7 +42,26 @@ impl TTY {
         Err(sys_common::unimpl())
     }
     pub fn get_winsize(&mut self) -> IoResult<(int, int)> {
-        Err(sys_common::unimpl())
+        unsafe {
+            #[repr(C)]
+            struct winsize {
+                ws_row: u16,
+                ws_col: u16,
+                ws_xpixel: u16,
+                ws_ypixel: u16
+            }
+
+            let mut size = winsize { ws_row: 0, ws_col: 0, ws_pixel: 0, ws_ypixel: 0 };
+            if funcs::bsd44::ioctl(0, 0x00005413, &mut size) == -1 {
```

Comment:
Can you move this literal into a `const` with a corresponding name that it has in C?